### ISSUE TITLE
npx create-nx-workspace fails on windows when path has space

### ISSUE THREAD

**Issue by sai-github** [nrwl/nx#4150]
## Current Behavior
After running the cmd at dir `C:\Users\username\Documents\my space\folder`
It creates the folders at `C:\Users\username\Documents\my` but throws error when opting for nxCloud

## Expected Behavior
Need to create the folder structure in mentioned folder even when path has space character

## Steps to Reproduce
- run `npx create-nx-workspace` inside a path then folder name has space character (I don't have nx installed globally)
- (shows no error when opting out of cloud)
- (creates workspace where the space character is encountered

### Failure Logs
(when nxCloud is opted in else no error is shown but the workspace is created at incorrect path)
```
(node:18512) UnhandledPromiseRejectionWarning: Error: spawnSync C:\WINDOWS\system32\cmd.exe ENOENT
    at Object.spawnSync (internal/child_process.js:1041:20)
    at spawnSync (child_process.js:607:24)
    at Object.execSync (child_process.js:652:15)
    at createApp (C:\Users\saied\AppData\Roaming\npm-cache\_npx\18512\node_modules\create-nx-workspace\bin\create-nx-workspace.js:437:25)
    at C:\Users\saied\AppData\Roaming\npm-cache\_npx\18512\node_modules\create-nx-workspace\bin\create-nx-workspace.js:100:29
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
(node:18512) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:18512) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

### Environment
Windows
(By executing `npm run nx report` inside the repo created at path `C:\Users\username\Documents\my`
```
  nx : Not Found
  @nrwl/angular : 10.4.4
  @nrwl/cli : 10.4.4
  @nrwl/cypress : 10.4.4
  @nrwl/eslint-plugin-nx : Not Found
  @nrwl/express : Not Found
  @nrwl/jest : 10.4.4
  @nrwl/linter : 10.4.4
  @nrwl/nest : 10.4.4
  @nrwl/next : Not Found
  @nrwl/node : 10.4.4
  @nrwl/react : Not Found
  @nrwl/schematics : Not Found
  @nrwl/tao : 10.4.4
  @nrwl/web : Not Found
  @nrwl/workspace : 10.4.4
  typescript : 4.0.5
```

**Comment by bekos** [comment]
@sai-github unfortunately I cannot reproduce your issue: 
![image](https://user-images.githubusercontent.com/551595/100328663-70af2c80-2fd5-11eb-99fe-aaa23326ca6e.png)

Please provide more info like if you use cmd/powershell/..., Node version, npm version etc

**Comment by sai-github** [comment]
```
node : v12.14.0
npm  : 6.13.4
```

I still get same error in cmd prompt and terminal inside vs code

I tried below from your screenshot and still got the same error
- `npx create-nx-workspace testspace-cloud`
- select empty project

which gave output as `new testspace-cloud  --preset="empty" --linter="eslint" --nxCloud --interactive=false --collection=@nrwl/workspace`, creates files in wrong dir and later fails with same failure log

I am not sure why its not reproducible, maybe there is something wrong with my system. You may close it or if you need some more information, I am happy to provide. I am working in a directory with no spaces and there is no issue.

**Comment by Tomerxp** [comment]
I got the same error on my machine using this path
C:\Users\Tomer\Desktop\projectname

Switched to C:\projectname and it now works
Not sure why it failed, maybe path was too long?

**Comment by bekos** [comment]
I have seen this happening, and seems to be related with the bash environment. Keeping it open.

**Comment by bekos** [comment]
This is fixed via 9abdb78a4ab44c8bee1ca8fb09c763c011dc1db3. We also now have e2e tests for windows to confirm this.

**Comment by maxisam** [comment]
Not sure if it is fixed. I just got the same error by creating a brand new workspace

>  NX   SUCCESS  Nx has successfully created the workspace.


———————————————————————————————————————————————


>  NX   ERROR  Something went wrong! v12.3.4

(node:37776) UnhandledPromiseRejectionWarning: Error: spawnSync C:\Windows\system32\cmd.exe ENOENT
    at Object.spawnSync (internal/child_process.js:1067:20)
    at spawnSync (child_process.js:602:24)
    at Object.execSync (child_process.js:647:15)
    at C:\Users\saml\AppData\Local\Yarn\Data\global\node_modules\create-nx-workspace\bin\create-nx-workspace.js:432:29
    at Generator.next (<anonymous>)
    at fulfilled (C:\Users\saml\AppData\Local\Yarn\Data\global\node_modules\create-nx-workspace\node_modules\tslib\tslib.js:114:62)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:37776) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:37776) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
Done in 468.10s.
